android-godeye/src/main/java/cn/hikyson/godeye/core/internal/modules/leakdetector/DefaultLeakRefInfoProvider.java

```diff
@@ -5,24 +5,33 @@
 import android.support.annotation.NonNull;
 import android.support.v4.app.Fragment;
 
+import java.util.HashMap;
+import java.util.Map;
+
 @Keep
 public class DefaultLeakRefInfoProvider implements LeakRefInfoProvider {
 
     @NonNull
     @Override
     public LeakRefInfo getInfoByActivity(Activity activity) {
-        return new LeakRefInfo(false, activity.getClass().getSimpleName());
```

Comment:
@ahhbzyz pageid is not supposed to  be here, can be null instead if you do not need this